hs_core/templates/pages/genericresource.html

```diff
@@ -324,7 +324,14 @@ <h4 class="modal-title">Confirm file deletion</h4>
                                         <table class="table table-striped">
                                             <tbody>
                                             <tr>
-                                                <th class="division-title">Authors</th>
+                                                <th class="division-title" colspan="5"><span>Authors </span>
+                                                    <div>
+                                                        <small class="text-muted division-title-caption">The people
```

Comment:
It looks like @dtarb  suggested the text needs to be:

"People or Organizations credited for creation of the intellectual content of the resource at a level that warrants authorship. This is synonymous with Creator in Dublin Core Metadata."